src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import CopyIcon from '../../../public/assets/icons/copy';
+import AddIcon from '../../../public/assets/icons/add';
+import ReloadIcon from '../../../public/assets/icons/reload';
+import Layout from '@/components/Layout';
+
+const Dashboard = () => {
+    const [url, getUrl] = useState('');
+    const [shortUrl, setUrl] = useState('');
+
+    function generateRandomString() {
+        const randomString = Math.random().toString(36).substring(2, 7);
+        return randomString;
+    }
+
+    return (
+        <Layout title="Home | URL Shortener">
+            <div className="flex flex-col justify-center items-center space-y-4 w-[90%] max-w-3xl md:w-[80%] lg:w-[80%] h-screen">
+                <h1 className="text-4xl text-white font-semibold">URL Shortener</h1>
+                <div className="flex flex-col md:flex-row justify-center items-center space-y-4 md:space-y-0 md:space-x-1 m-4 w-[100%]">
+                    <input
+                        type="text"
+                        placeholder="Enter the URL"
+                        className="w-full md:w-[80%] bg-gray-200 md:rounded-l-2xl p-4 md:rounded-none"
```

Comment:
- Instead of using a % width can we use a definite value here ?

Reply:
updated